static/src/components/setting/index.vue

```diff
@@ -18,6 +18,12 @@
             v-for="field in category.fields"
             :key="field.name"
           >
+            <multiselect
```

Comment:
Since all users would be able to see this, we should add label letting them know they need admin access to change this value, or hide it for non admins.